services/spar/test-integration/Test/Spar/APISpec.hs

```diff
@@ -908,6 +908,20 @@ specCRUDIdentityProvider = do
           `shouldRespondWith` ((== 200) . statusCode)
         callIdpGet (env ^. teSpar) (Just owner) idpid
           `shouldRespondWith` ((== idpmeta') . view idpMetadata)
+      it "updates IdP metadata and creates a new IdP with the first metadata" $ do
+        env <- ask
+        (owner, _) <- call $ createUserWithTeam (env ^. teBrig) (env ^. teGalley)
```

Comment:
```suggestion
        (owner, _) <- createUserWithTeam' (env ^. teBrig) (env ^. teGalley)
```

(unless i'm missing something, which is very possible.)

Reply:
The function `createUserWithTeam'` does not exist in spar.